src/components/common/ProgressBar.tsx

```diff
@@ -0,0 +1,137 @@
+import styled from '@emotion/styled';
+import React from 'react';
+import Text from './Text';
+
+interface Props {
+  stepCurrentIdx: number;
```

Comment:
```suggestion
  stepCurrentIndex: number
```

`stepCurIdx` 이런식으로 되거나, `stepCurrentIndex` 요런식으로 해야 통일성있을 것 같아요!